README.md

```diff
@@ -9,6 +9,14 @@ curl -O https://raw.githubusercontent.com/lukaswrz/arch-install/main/arch-instal
 bash arch-install.bash
 ```
 
+You can automatically install your dotfiles by setting paramters to the script
+
+`Usage: $0 [-h] [-g repository] [argv...]`
+
+For example:
+
+`$ bash arch-install.bash -g https://github.com/rathmerdominik/testrepofordotfiles -- bash testscript.bash `
```

Comment:
The usage above suffices IMO.

Reply:
I think having a small example is a good thing. better a bit more than less IMO.
But because this is your repository i will follow that and will remove it